server/service/integration_mdm_test.go

```diff
@@ -2761,6 +2764,11 @@ func (s *integrationMDMTestSuite) TestEnqueueMDMCommand() {
 	ctx := context.Background()
 	t := s.T()
 
+	// Skip worker jobs to avoid running into timing issues with this test.
+	// We can manually run the jobs if needed with s.runWorker().
+	s.skipWorkerJobs = true
```

Comment:
Not doing a full review, but just wanted to highlight that this is racy, as there's a write here and a read in another goroutine. We should use an atomic.Bool instead.

Reply:
Done!